ui/static/js/app.js

```diff
@@ -2,6 +2,16 @@
 (function() {
 'use strict';
 
+// Polyfill for Internet Explorer
+if (!String.prototype.endsWith) {
```

Comment:
Miniflux supports only ES6 browsers, you can remove this polyfill.

Reply:
Removed.